server/src/data-layer/services/EventService.ts

```diff
@@ -25,6 +26,24 @@ class EventService {
     return result.data()
   }
 
+  /**
+   * Fetches all events that have a start date in the future.
+   * Note that "active" means any event that is still upcoming, or "ready" to signup.
```

Comment:
does that mean we need to check the `end_date` hasn't expired?

Reply:
that raises another question, we might need an extra `event_date` for when the event actually is

Reply:
> does that mean we need to check the `end_date` hasn't expired?

No, not necessarily. The `start_date` variable is supposedly ≥ `end_date` so essentially any `start_date` that is ≥ today would be a "scheduled" event. 

> that raises another question, we might need an extra `event_date` for when the event actually is

But couldn't events be lasting several days?

Reply:
I think my reasoning is that the signup for the event is different from the actual event date

Reply:
> I think my reasoning is that the signup for the event is different from the actual event date

Ahhh right, I supposed that an event start_date & end_date was the actual planned dates for the signups and that they close reservations from the point of start_date